amgut/lib/locale_data/american_gut.py

```diff
@@ -598,7 +602,7 @@
     'ENTER_STATE': 'Please enter your state',
     'ENTER_ZIP': 'Please enter your zip',
     'ENTER_COUNTRY': 'Please enter your country',
-    'REQUIRED_ZIP': 'Your zip must consist of at least 5 characters',
+    'REQUIRED_ZIP': 'Your zip must be 10 or fewer characters',
```

Comment:
Where is this requirement implemented? It does not appear to be enforced in the database schema definition anywhere. The only limitation I can find is on the `ag_map_markers` table where it is limited to 20.